includes/tracks/events/class-wc-orders-tracking.php

```diff
@@ -21,6 +21,26 @@ public function init() {
 		// WC_Meta_Box_Order_Actions::save() hooks in at priority 50.
 		add_action( 'woocommerce_process_shop_order_meta', array( $this, 'track_order_action' ), 51 );
 		add_action( 'load-post-new.php', array( $this, 'track_add_order_from_edit' ), 10 );
+		add_filter( 'woocommerce_shop_order_search_results', array( $this, 'track_order_search' ), 10, 3 );
+	}
+
+	/**
+	 * Send a track event when on the Order Listing page, and search results are being displayed.
+	 *
+	 * @param array  $order_ids Array of order_ids that are matches for the search.
+	 * @param string $term The string that was used in the search.
+	 * @param array  $search_fields Fields that were used in the original search.
+	 */
+	public function track_order_search( $order_ids, $term, $search_fields ) {
+		$screen = get_current_screen();
```

Comment:
`woocommerce_shop_order_search_results` runs in many places since it runs on the CRUD, and also by `wc_order_search()` on the REST API and other places.
I'm aware that we only load tracking classes on the admin, still feels like probably better check if `get_current_screen()` exists to avoid any problems in the future.

Reply:
check added in 94bf2eb - great catch 🙇